### PR TITLE
Brand user locations

### PR DESCRIPTION
**Pull request by mmadariaga**
Expose user locations on brand API. Added methods:
- /friends/status
- /residential_devices/status
- /retail_accounts/status
- /terminals/status

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/rest/brand/features/provider/friend/getFriendStatus.feature [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Feature: Retrieve friends status
+  In order to manage friends status
+  As a client admin
```
Comment (Kaian) [inline web/rest/brand/features/provider/friend/getFriendStatus.feature]: As a brand admin

File: web/rest/brand/features/provider/residentialDevice/getResidentialDeviceStatus.feature [hunk 2]
```diff
@@ -0,0 +1,31 @@
+Feature: Retrieve residential devices status
+  In order to manage residential devices status
+  As a client admin
```
Comment (Kaian) [inline web/rest/brand/features/provider/residentialDevice/getResidentialDeviceStatus.feature]: As a brand admin

File: web/rest/brand/features/provider/retailAccount/getRetailAccountStatus.feature [hunk 3]
```diff
@@ -0,0 +1,32 @@
+Feature: Retrieve retail accounts status
+  In order to manage retail accounts status
+  As a client admin
```
Comment (Kaian) [inline web/rest/brand/features/provider/retailAccount/getRetailAccountStatus.feature]: As a brand admin

File: web/rest/brand/features/provider/terminal/getTerminalStatus.feature [hunk 4]
```diff
@@ -0,0 +1,45 @@
+Feature: Retrieve terminals status
+  In order to manage terminals status
+  As a client admin
```
Comment (Kaian) [inline web/rest/brand/features/provider/terminal/getTerminalStatus.feature]: As a brand admin

**Pull request merged**